packages/components/psammead-story-promo-list/src/index.stories.jsx

```diff
@@ -112,3 +112,32 @@ storiesOf('Components|StoryPromo/StoryPromoList', module)
       </Grid>
     );
   });
+
+storiesOf('Components|StoryPromo/StoryPromoListBase', module).add(
```

Comment:
It'd be great if we could add a `border` true/false knob to these stories.

Reply:
@simonsinclair done!

Reply:
This doesn't seem to change anything when I check it in storybook: https://5d28eb5ee163f6002046d6fb-erzsbwuwtq.chromatic.com/?path=/story/components-storypromo-storypromolistbase--default